release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -210,6 +210,35 @@ For more information, see xref:../post_installation_configuration/cluster-capabi
 
 * With this update, in the *Add+* page of the developer perspective, a new menu added to share project and Helm Chart repositories that allows to add or remove users to the project.
 
+[id="ocp-4-11-dynamic-plug-in-updates"]
+==== Dynamic plug-in updates
+
+With this update, you can use the new `console.openshift.io/default-i18next-namespace` annotation to determine if the `ConsolePlugin` contains localization resources. If the annotation is set to `"true"`, the localization resources from the i18n namespace named after the dynamic plug-in, are loaded. If the annotation is set to any other value or is missing on the `ConsolePlugin` resource, localization resources are not loaded.
```

Comment:
the annotation is `console.openshift.io/use-i18n`

Reply:
Updated! Thanks for clarifying!